src/App.tsx

```diff
@@ -1,9 +1,15 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = moviesFromServer.filter(
+    movie => movie.title.toLowerCase().includes(query.toLowerCase())
```

Comment:
```suggestion
    movie => movie.title.toLowerCase().includes(query.toLowerCase().trim())
```
<img width="648" alt="image" src="https://user-images.githubusercontent.com/55802630/184600774-52d14cb7-6994-4ff0-b8f9-e14be7fa6c0a.png">